pyconcz_2016/speakers/migrations/0002_talk.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.3 on 2016-09-17 02:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('speakers', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Talk',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=200)),
+                ('description', models.CharField(blank=True, max_length=255)),
+                ('speaker', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to='speakers.Speaker')),
```

Comment:
Alrighty, I've updated the PR with support for multiple people per talk, along with some other minor tweaks. Could you review again @tricoder42?